languages/common-lisp/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,61 @@
+You! Yes you, brave lisper! [Lenny the lisp alien][alien] needs your help! He
+wants to go hang out with all the other lisp aliens, but his mom has told him
+that he's not going anywhere until his room is clean.
+
+The issue is, Lenny can hardly see the floor! He can't tell his atoms from his
+conses, his symbols from his keywords, his `car`s from his `cdr`s!
+
+Can you help Lenny sort this mess out? Don't forget [your parentheses][xkcd],
+you'll be needing them!
+
+## 1. Refresh Lenny on symbols
+
+Before Lenny can get started, he needs a refresher on what symbols and keywords
+look like – it's been a while since he last tidied up... Help Lenny out by
+defining two functions: one that returns a symbol, and another that returns a
```

Comment:
The instructions for other tracks explicitly name the function to be defined in the text too. See [this example](https://github.com/exercism/v3/blob/fsharp/strings/after-syntax/languages/fsharp/exercises/concept/strings/.docs/instructions.md#1-get-message-from-a-log-line).